week10/.gitignore

```
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
```

```suggestion
.env
.env.local
```

.gitignore에 .env가 추가되어 있지 않아서 깃허브에 환경변수 파일이 업로드 되었네요!! 지금은 단순 세미나 과제 서버 주소라서 큰 문제는 없지만 추후 프로젝트 진행하실 때 민감한 정보(API key, secret)가 업로드 되지 않도록 반드시 .gitignore 재확인하시고, git push 전 `git status` 명령어로 변경사항에 .env 파일이 포함되지 않았는지 확인하시는 것을 추천합니다.